packages/web-frontend/src/components/View/View.js

```diff
@@ -26,7 +26,7 @@ const viewHasData = viewContent => {
   }
 
   // If all segments of a pie chart are "0", display the no data message
-  if (chartType === CHART_TYPES.PIE && data && data.every(segment => segment.value === 0)) {
+  if (chartType === ChartType.PIE && data && data.every(segment => segment.value === 0)) {
```

Comment:
Just double checking, should this be `ChartType.Pie`?

Reply:
Yes it should, thanks